src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -99,75 +99,6 @@ class GcsPlacementGroupSchedulerInterface {
   virtual ~GcsPlacementGroupSchedulerInterface() {}
 };
 
-class GcsScheduleStrategy {
```

Comment:
To be clear:
All placement group related strategies are moved out from `gcs_placement_group_scheduler.h/.cc` to `bundle_scheduling_policy.h/.cc`